README.md

```diff
@@ -15,6 +15,12 @@ on how to do that, including how to develop and test locally and the versioning
 
 _Note: 1.28.0 and later require Gradle 7_
 
+### 1.39.0-refactor-SNAPSHOT
+*Released*: TBD
+(Earliest compatible LabKey version: 22.9)
```

Comment:
I guess it's still technically compatible with 22.9 since users can provide the 22.9 version of the path via the setting property.